BizHawk.Emulation.Cores/Consoles/Atari/2600/Mappers/m4A50.cs

```diff
@@ -89,17 +89,11 @@ private byte[] RomImage
 
 		public override bool HasCartRam => true;
 
-		public override ByteBuffer CartRam => _ram;
-
-		public override void Dispose()
-		{
-			_ram.Dispose();
-			base.Dispose();
-		}
+		public override byte[] CartRam => _ram;
 
 		public override void SyncState(Serializer ser)
 		{
-			ser.Sync("cartRam", ref _ram);
+			ser.Sync("cartRam", ref _ram, false);

```

Comment:
Does it make sense to add another overload for `Sync(string, ref T[])` so we don't need to add this `false` parameter in many places?

Reply:
Yeah, I don't think that's in scope for this PR, but it's something I could look at, I think it would also reduce the number of overloads we have for this method

Reply:
Maybe it should be in scope for this PR, because if done now it would make this PR feel more natural/smooth?  Since diffs like this line wouldn't even exist, there'd be less noise in the git history and you'd convey the intent of this PR better:  There should be no change in savestate functionality.

Your call though, I have no strong feelings about it.